service/http.balancer/src/test/java/org/kaazing/gateway/service/http/balancer/WsebBalancerIT.java

```diff
@@ -0,0 +1,85 @@
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.kaazing.gateway.service.http.balancer;
+
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.gateway.server.test.GatewayRule;
+import org.kaazing.gateway.server.test.config.GatewayConfiguration;
+import org.kaazing.gateway.server.test.config.builder.GatewayConfigurationBuilder;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+import java.net.URI;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.rules.RuleChain.outerRule;
+
+public class WsebBalancerIT {
+
+    private final K3poRule robot = new K3poRule();
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(10, TimeUnit.SECONDS));
+
+    private final GatewayRule gateway = new GatewayRule() {
+        {
+            GatewayConfiguration configuration = new GatewayConfigurationBuilder()
+                    // balancer service to echo
+                    .service()
+                        .type("balancer")
+                        .accept(URI.create("ws://gateway.example.com:8001/echo"))
+                        .acceptOption("tcp.bind", "localhost:8001")
+                        .crossOrigin()
+                            .allowOrigin("*")
+                        .done()
+                    .done()
+                    // echo service
+                    .service()
+                        .type("echo")
+                        .accept(URI.create("ws://node.example.com:8001/echo"))
+                        .balance(URI.create("ws://gateway.example.com:8001/echo"))
+                        .acceptOption("tcp.bind", "localhost:8001")
+                        .crossOrigin()
+                            .allowOrigin("*")
+                        .done()
+                    .done()
+            .done();
+
+            init(configuration);
+        }
+    };
+
+
+    @Rule
+    public TestRule chain = outerRule(robot).around(gateway).around(timeout);
+
+    @Test
+    @Specification("wse.balancer.request")
+    public void wseBalancerRequest() throws Exception {
```

Comment:
Usually we try to make test the names reflect the expected outcome sent immediately obvious what is testing for. How about balancerRequestShouldRespondWithRedirect?